lib/course_planner_web/router.ex

```diff
@@ -29,6 +35,13 @@ defmodule CoursePlannerWeb.Router do
   end
 
   scope "/" do
+    pipe_through :public_api
+
+    resources "/api/sessions", CoursePlannerWeb.Auth.Api.V1.JsonSessionController,
```

Comment:
The API versioning should be exposed to the client, so the `v1` should be either in the `scope` or in the route.

Reply:
if we agree to have the V1 in the name of the module then i suggest to have it as scope

Reply:
Let's remove the module name an have the `/api/v1` on the scope.

Reply:
done